t0/seqio_tasks/experiment_D4.csv

```diff
@@ -0,0 +1,73 @@
+HF_name,subset,task_by_convention,format,comment,seed_paper,do_train,do_eval,train_size,adjusted_train_size,metric,multiple correct answer,Paper link,non_linguistic_knowledge,Imported Task Name,imported category,input_length,_human_skill,Domain,Reference,done

```

Comment:
Maybe we need to remove https://github.com/bigscience-workshop/t-zero/blob/master/evaluation/template_list.py as it should be included in this? In fact this file should probably be shared to non seqio users, ie people who would want to run this on pytorch for example.